packages/babel-helper-check-duplicate-nodes/src/index.ts

```diff
@@ -0,0 +1,48 @@
+// https://github.com/babel/babel/blob/d383659ca6adec54b6054f77cdaa16da88e8a171/packages/babel-helper-transform-fixture-test-runner/src/index.js#L128
+export default function checkDuplicateNodes(babel, ast) {
```

Comment:
Q: Why don't we just make `@babel/types` a dependency, rather than passing it as a parameter?

Reply:
I am good either way. I guess since we have renamed the package anyway, we can also change the interface a bit?

Reply:
Yeah, it's going to be a different major version and a different name. We could add a `Migration from babel-check-duplicated-nodes` section to its docs, or `console.warn` if `arguments.length === 2`.